### PR TITLE
Desy dev qcd estimation v2

### PR DESCRIPTION
**Pull request by jmalvaso**

### CODE REVIEW

File: httcp/calibration/main.py [hunk 1]
```diff
@@ -45,11 +45,11 @@ def main(self: Calibrator, events: ak.Array, **kwargs) -> ak.Array:
     events = set_ak_column_f32(events, "PuppiMET.pt_no_jec", events.PuppiMET.pt)
     events = set_ak_column_f32(events, "PuppiMET.phi_no_jec", events.PuppiMET.phi)
 
-    print("Performing Jet Energy Correction...")
-    events = self[jec](events, **kwargs)
```
[the remark this answers is not in the document]
Comment (hephysicist) [reply httcp/calibration/main.py:49]: Should it be like so, or we need to reinclude it?

[the remark this answers is not in the document]
Comment (jmalvaso) [reply httcp/calibration/main.py:49]: Definitely a mistake on my part, I'll include them again with the next commit. @hephysicist it is mandotory to apply L1L2L3Rec correction to Jets and then propagate them to met, so we have to include jec always from now on

[the remark this answers is not in the document]
Comment (anigamova) [reply httcp/calibration/main.py:49]: I think we want to run with jet energy corrections now, it is probably a remnant from your branch @hephysicist , but we want them back. It is just that I'm giving heads up that the production which was submitted earlier today is probably bugged

[the remark this answers is not in the document]
Comment (jmalvaso) [reply httcp/calibration/main.py:49]: @anigamova if you lauched the processing of the mutau or etau preEE using the branch desy_dev all the steps untill the production ones are fine, so I didn't waste time (as I did :) )

[the remark this answers is not in the document]
Comment (hephysicist) [reply httcp/calibration/main.py:49]: Got it. But I didn't push anything to desy_dev for quite a while. 
Can we accept this commit, so I can by hand include jec and reprocess everything during the night?

[the remark this answers is not in the document]
Comment (jmalvaso) [reply httcp/calibration/main.py:49]: @hephysicist I think it's a nice idea, I can accept the pull request, you fix this jec thing and then you can process safely the samples. Btw it is my fault that I cherry picked your commit without checking... sorry!

[the remark this answers is not in the document]
Comment (anigamova) [reply httcp/calibration/main.py:49]: no, not working at all :-) 
 I will push a fix, give me a minute

[the remark this answers is not in the document]
Comment (jmalvaso) [reply httcp/calibration/main.py:49]: I can also push that commit actually, I'm just too lazy, thank you @anigamova !

[the remark this answers is not in the document]
Comment (hephysicist) [reply httcp/calibration/main.py:49]: Sorry people for pinging you outside of working hours...

**Pull request merged**